### ISSUE TITLE
More detailed information

### ISSUE THREAD

**Issue by sXRaider** [MicrosoftDocs/memdocs#4045]
The page is good already, but a lot of details of how exactly the process works and how we can follow a bit via logs on the devices itself is really missing. Can more information be provided for IT Admins please? Now we are just guessing.

Also: shouldn't there be mentionned that if GPO has setting WSUS for driver updates, that it should be changed to Windows Update for this to work (We guessed this)

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 7928e48c-63f2-e459-db87-bb62eb5501b0
* Version Independent ID: 7928e48c-63f2-e459-db87-bb62eb5501b0
* Content: [Learn about Windows Driver updates policy for Windows 10 Windows 11 devices in Intune](https://learn.microsoft.com/en-us/mem/intune/protect/windows-driver-updates-overview)
* Content Source: [memdocs/intune/protect/windows-driver-updates-overview.md](https://github.com/MicrosoftDocs/memdocs/blob/main/memdocs/intune/protect/windows-driver-updates-overview.md)
* Service: **microsoft-intune**
* Sub-service: **protect**
* GitHub Login: @Brenduns
* Microsoft Alias: **brenduns**

**Comment by yogkumgit** [comment]
@Brenduns Please help us in resolving this issue. Thanks

**Comment by Brenduns** [comment]
@sXRaider   Thanks for the feedback.  As to MDM based policies vs GPO's , that is a much larger discussion than this single topic. However, generally, MDM based policies and settings should override GPOs if both are in place.  However, I've recently discovered https://learn.microsoft.com/en-us/windows/client-management/mdm/policy-csp-controlpolicyconflict (ControlPolicyConflict) which seems to also affect how that plays out. I've not seen nor heard of this CSP before today.  So while that is interesting, I've got to fall back on my current understanding of MDM winning over GPO based configurations - as part of a general manageability push or direction for Windows devices. (Though here I am also leaning heavy on our dev partners, and don't have deeper background on this subject.) 

As to the other question about how the process works and to follow via logs on the devices, check out the FAQ for a few bits of that.  Some details in the FAQ section can shed light on the WUfB ds Synchronization frequency, how quickly pausing an update is enacted, and driver inventory times.  Beyond the hints of light provided by the FAQ, we'll have to investigate what would be most useful to provide, and then schedule time to draft that content.  As the product team had no initial thoughts on this subject, nor provided such details, it would be future work we’ll need to investigate before being able to turn into content.   But your asks is so noted!
